packages/ipfs-http-client/src/files/read.js

```diff
@@ -13,7 +15,7 @@ module.exports = configure(api => {
       searchParams: options
     })
 
-    for await (const chunk of toIterable(res.body)) {
```

Comment:
@hugomrdias It's interesting that there are still some methods where this `toIterable(res.body)` pattern is still around.